src/pages/quiz/summmary/QuizPage.tsx

```diff
@@ -54,6 +55,9 @@ const QuizPage = ({quizId, workspaceId}: { quizId: string, workspaceId: string }
         httpClient.getQuizQuestions(quizId)
             .then(setQuestions)
             .catch(console.error);
+        httpClient.getIncorrectQuizQuestions(quizId)
```

Comment:
Add this also to `useFocusEffect` function to reload incorrect questions each time user enters this view. We need that to disable button after user solves quiz to 100%
I know, react native is weird 👀